---
std_msgs/Header header
sensor_msgs/Image image     # Image at time of recognition; returned object data corresponds with this image
Object[] objects            # Detected (labeled) objects
